src/app/routes/utils/regex/utils/index.js

```diff
@@ -182,7 +182,7 @@ export const getIdxPageRegex = () => {
   let idxPageRegex = idxRegex;
 
   if (isLive()) {
-    idxPageRegex = `${idxRegex}|persian/afghanistan`;
+    idxPageRegex = `${idxRegex}`;
   }
 
   return `/:idx(${idxPageRegex}):amp(${ampRegex})?`;
```

Comment:
We can just use the `idxRegex` value as declared above - no need to use a separate variable

```suggestion

  return `/:idx(${idxRegex}):amp(${ampRegex})?`;
```